Docs OT-2 system description revisions

jwwojak opened the pull request:
# Overview

This adds a System Description section to the revised OT-2 manual. This section should contain 3 files:

- components
- pipettes
- robot specs

Sandbox: https://sandbox.docs.opentrons.com/docs-ot2-system-description-revisions/ot-2/system-description/

RTC-859

## Test Plan and Hands on Testing

Have the usual suspects evaluate the content.

## Changelog

When ready, should be 3 files in a System Description section.

## Review requests

Does the text make sense?
Does the organization make sense?
Are the images clear and relevant?

## Risk assessment

Low - moderate due to complexity of changes.

jwwojak committed:
System description-starting

jwwojak committed:
Add index page and placeholder text

jwwojak committed:
Lunch commit

Intro, main section, deck, outline for adding stuff today

jwwojak committed:
Quick save

jwwojak committed:
Changing branch save

jwwojak committed:
end of day commit

jwwojak committed:
EOW save

jwwojak committed:
mid-day commit

jwwojak committed:
another commit

jwwojak committed:
Good draft for review

For this section, it seems complete.

jwwojak committed:
Add apostrophe and revise comment

jwwojak committed:
More organization

jwwojak committed:
Pipettes page

Table specs from website, flow rates linked to API docs.

Other commits may have the GEN2 pipette images. Adding placeholder reminder text to check. Not duplicating images in this commit/PR just in case.

jwwojak committed:
Premeeting commit

Pipette text revisions and formatting

jwwojak committed:
Add link

ecormany committed:
Merge branch 'docs-ot2-manual-revisions' into docs-ot2-system-description-revisions

jwwojak committed:
Update ot2-power.png

Commit and will re-add power symbol that's being deleted.

jwwojak committed:
Fixing some merge problems

And unique header warnings.

jwwojak committed:
robot specs table

how big
how heavy
how much power
wifi
etc

jwwojak committed:
intro index page for section

jwwojak committed:
Revising index intro to section

jwwojak committed:
Reviewing and revising

Reading it and making changes.

jwwojak committed:
Some review changes on Friday

Giving it another read after an interval away. Making changes.

ecormany commented on docs/ot-2/docs/system-description/robot-components.md:
```diff
@@ -0,0 +1,120 @@
+
+- LED strips within the enclosure that provide software-controllable ambient lighting.
+- A built-in 2-megapixel camera that can take still or video images of the deck and working area.
+- A front LED that indicates the robot's status. It turns solid blue when your OT-2 powered on and fully booted up.
```
```suggestion
- A front LED that indicates the robot's status. It turns solid blue when your OT-2 is powered on and fully booted up.
```

ecormany commented on docs/ot-2/docs/system-description/robot-components.md:
```diff
@@ -0,0 +1,120 @@
+
+## OT-2 deck and working area
+
+The deck is the machined aluminum surface on which automated science protocols are executed. It provides 11 ANSI/SLAS-compliant slots that can hold labware, modules, and consumables. These deck slots are numbered 1 to 11. A removable trash bin occupies its own special slot in the  rear right corner of the deck.
```
stray space
```suggestion
The deck is the machined aluminum surface on which automated science protocols are executed. It provides 11 ANSI/SLAS-compliant slots that can hold labware, modules, and consumables. These deck slots are numbered 1 to 11. A removable trash bin occupies its own special slot in the rear right corner of the deck.
```

ecormany commented on docs/ot-2/docs/system-description/robot-components.md:
```diff
@@ -0,0 +1,120 @@
+
+The working area is the physical space on and above the deck that is accessible for pipetting. Labware placed in slots 1–11 are in the working area.
+
+The OT-2 deck slides in and out of the enclosure. You can pull it out part way, or remove it completely, to more easily place labware or clean the deck.
```
I wouldn't suggest removing the deck for labware placement. If you pull out the OT-2 deck, you should recalibrate it. If you need to recalibrate it, you have to remove all the labware.

jwwojak replied:
Will revise to just mention cleaning.

ecormany commented on docs/ot-2/docs/system-description/robot-components.md:
```diff
@@ -0,0 +1,120 @@
+  <tbody>
+    <tr>
+      <td><img src="../../images/power-symbol.png" alt="power" width="50%"></td>
+      <td>Power light<br>The light should stay on and remain solid if the OT-2 is turned on and initialized.</td>
```
Could also do three columns (Icon, Status, Description) instead of the internal breaks in these cells.

jwwojak replied:
Will give that a try.

ecormany commented on docs/ot-2/docs/system-description/robot-components.md:
```diff
@@ -0,0 +1,120 @@
+| Characters | Category | Description |
+|----|----|----|
+| OT2 | Model | The robot is an OT-2. |
+| CEP | Version | A code for the production version of the robot. |
```
I've never figured out what "CEP" actually stands for.

jwwojak replied:
Shrouded by the distant mists of time lie the origins of the OT-2 serial number.

<img width="500" height="956" alt="Image" src="https://github.com/user-attachments/assets/99048656-476b-4e72-a01e-8b494d105e9f" />

ecormany commented on docs/ot-2/docs/system-description/pipettes.md:
```diff
@@ -0,0 +1,194 @@
+
+## Installation
+
+For instructions on installing pipettes, see the [Instrument Installation and Calibration section](../installation/instruments.md).
```
Since this is just one line, I would suggest removing the header.

We don't need it for search, either: 

<img width="677" height="434" alt="Image" src="https://github.com/user-attachments/assets/830e8395-2557-48fb-8363-78d37779137a" />

jwwojak replied:
Right! Will remove.

ecormany commented on docs/ot-2/docs/system-description/pipettes.md:
```diff
@@ -0,0 +1,194 @@
+
+To discard tips (or return them to their rack), the pipette ejector mechanism pushes the tips off of the nozzles.
+
+## OT-2 pipette specifications¶
```
```suggestion
## OT-2 pipette specifications
```

ecormany commented on docs/ot-2/docs/system-description/pipettes.md:
```diff
@@ -0,0 +1,194 @@
+      <th>%D</th>
+      <th>Systematic Error (µL)</th>
+      <th>%CV</th>
+      <th>Random Error (µL)</th>
```
I think it would be good to use two `<th>` and a colspan here to achieve a similar effect as in the source material.

<img width="317" height="53" alt="Image" src="https://github.com/user-attachments/assets/c02d5600-fac3-4d89-a7b3-3da0d7ce5a4a" />

jwwojak replied:
Can do.

jwwojak replied:
We might want to keep a header and column for total volumes. That gives context to the accuracy and precision amounts.

ecormany commented on docs/ot-2/docs/system-description/specs.md:
```diff
@@ -0,0 +1,85 @@
+    </tr>
+    <tr>
+        <td><strong>Weight</strong></td>
+        <td>The OT-2 weighs 48 kg (105 lbs).</td>
```
Department of Redundancy Dept.
```suggestion
        <td><strong>Dimensions</strong></td>
        <td>66 cm x 57 cm x 63 cm (≈ 26" x 22.5" x 25")</td>
    </tr>
    <tr>
        <td><strong>Weight</strong></td>
        <td>48 kg (105 lbs)</td>
```

jwwojak replied:
Oops.

Belt, suspenders, zip ties, rope, ratchet straps. Will fix.

ecormany commented on docs/ot-2/docs/system-description/specs.md:
```diff
@@ -0,0 +1,85 @@
+            <ul>
+                <li>Windows 10 or later</li>
+                <li>macOS 10.10 or later</li>
+                <li>Ubuntu 12.04 or later</li>
```
These are out of date. Latest reqs are on the app download page.

ecormany replied:
I looked this up recently and reqs changed at app version 8.3, if you want to include both.

jwwojak replied:
https://opentrons.com/ot-app

- Windows 10 or later
- macOS 10.16 or later
- Ubuntu 20.04 or later

ecormany reviewed (commented):
First pass of comments

jwwojak committed:
Apply suggestions from code review

Committing reviewer recommendations. Will implement other changes and commit separately.

Co-authored-by: Ed Cormany <edward.cormany@opentrons.com>

jwwojak committed:
Apply suggestion from @ecormany

Co-authored-by: Ed Cormany <edward.cormany@opentrons.com>

jwwojak committed:
Making reviewer recommended changes

ecormany reviewed (approved):
Looks good, no reason not to get this into your main manual branch at this point.
